lib/ts/supervisor.ex

```diff
@@ -26,6 +26,12 @@ defmodule TS.Supervisor do
 
 	args
 
+	###Function of Supervisor
+	The module TS.Supervisor is the top level supervisor for the project. The 
+	children of a Supervisor can be workers or supervisors. TS.Supervisor spawns
+	three children, the Network Constructor (worker), the API (worker)
+	and a Station Supervisor (Supervisor). 
+
 	### Strategies
```

Comment:
comments in the deleted file api_supervisor.ex are very helpful. They are generic and hence can be copied to supervisor.ex. Please do copy them.